pkg/dashboard/assets/js/overview.js

```diff
@@ -0,0 +1,431 @@
+            const attributeIndex = _.findIndex(groupBy,function(o) { return o.name == name; });
+
+            if (attributeIndex === -1) {
+                groupBy.push({
+                    name: name,
+                    count: 1
+                });
+                //console.log('adding attribute :' + name);
+            }else{
+                groupBy[attributeIndex].count +=1;
+            }
+        }
+    }
+    groupBy = _.sortBy(groupBy, 'count').reverse();
+    //console.log(groupBy);
+
+    $("#group-by-dropdown").empty();
+    for(let i=0;i<groupBy.length;i++){
+        const optionName = groupBy[i].name + ' (' + groupBy[i].count + ')';
+        const optionValue = groupBy[i].name;
+
+        $("#group-by-dropdown").append(new Option(optionName, optionValue));
+    }
+    $("#group-by-dropdown").val(defaultGroupBy);
+}
+
+function setListOfItemsByGroup() {
+
+    const groupBy = $("#group-by-dropdown").val();
+    //console.log(`[] grouping by `+ groupBy);
+
+
+    let results = [];
+
+    // scan images
+    for(let i=0;i<window.auditData.Results.length;i++) {
+        const image = window.auditData.Results[i];
+
+        const attributeIndex = _.findIndex(image.attributes, function (o) {
+            return o.startsWith(groupBy+ ':');
+        });
+
+        // if image does not have the attribute, skip
+        if (attributeIndex === -1) continue;
+
+        const attributeValue = image.attributes[attributeIndex].split(':')[1];
+
+        // check if the image with the attribute value exists under the group
+        const imageIndex = _.findIndex(results, function (o) {
+            return o.title === attributeValue;
+        });
+
+        if (imageIndex === -1) {
+            // create new image under group
+            results.push({
+                title: attributeValue,
+                images: [image]
+            });
+        }else {
+            // add existing image under group
+            results[imageIndex].images.push(image);
+        }
+
+    }
+
+
+    // render images
+    $("#results").empty();
+    for(let i=0;i<results.length;i++){
+        const resultGroup = results[i];
+
+        window.counters = {
+            "CRITICAL": 0,
+            "MEDIUM": 0,
+            "NOISSUES": 0,
+            "NODATA": 0
+        }
+
+        let sublist = `<ul>`;
+        for(let j=0;j<resultGroup.images.length;j++) {
+            const rowText = parseSeverities(resultGroup.images[j]);
+            sublist += '<li>';
+            sublist += ' <div style="float:right;">' + rowText + '</div>';
+            if(rowText === 'No Issues') {
+                sublist += '<i class="fas fa-check noissues-icon"></i>';
+            }else if (rowText === 'No Data') {
+                sublist += '<i class="fas fa-times nodata-icon"></i>';
+            }else {
+                sublist += '<i class="fas fa-exclamation-triangle warning-icon"></i>';
+            }
+            sublist += shortenImageName(resultGroup.images[j].image);
```

Comment:
Could you also add a link to a single-image page?

Reply:
done.